docs/Caching.md

```diff
@@ -0,0 +1,158 @@
+# Caching
+
+Web platform apps cache data at a few different levels in order to improve UI
+responsiveness. There are two primary types of content that are cached:
+
+1. REST API responses
+2. Static assets (files)
+
+## Overview
+
+![Cache diagram](https://user-images.githubusercontent.com/1885153/27892985-d75546fe-6256-11e7-8272-66252caa5c85.png)
+
+([from web platform diagrams](https://docs.google.com/presentation/d/1c8jf8UtGa81Ay4oqlbHfoP4Ile2VkiRVrCMVag6cRBQ/#slide=id.g1f921bb8da_0_0))
+
+## 1. API responses
+
+Currently the Meetup REST API does not provide a
+[`Cache-Control` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control)
+(e.g. `max-age`), so the 'raw' API responses cannot be directly cached at the
+app server or browser level - the app must re-fetch every API request from the 
+browser to ensure that it delivers the requested data.
+
+### CDN
+
+The CDN does not currently cache any API responses. [See the 'Future'
+discussion](#future) for potential changes to the behavior.
+
+### App server
+
+The app server does not currently cache any API responses. [See the 'Future' 
+discussion](#future) for potential changes to the behavior.
+
+### Browser
+
+The browser uses
```

Comment:
I'll add more info about how the cache is purged - it basically lasts as long as the logged-in session lasts. It can be manually purged in a similar way to clearing cookies, so I'll add that info